.azure-pipelines/pr_test_scripts.yaml

```diff
@@ -797,6 +797,122 @@ onboarding_t1_multi_asic:
   - pfcwd/test_pfcwd_function.py
   - pfcwd/test_pfcwd_timer_accuracy.py
 
+t2:
```

Comment:
We are using impact area based PR testing, the test scope is determined automatically. Is this hard coded list still required?